assets/xsl-stylesheets/xml-sitemap.xsl

```diff
@@ -80,9 +81,10 @@
                     <table id="sitemap" cellpadding="3">
                         <thead>
                             <tr>
-                                <th width="75%">URL</th>
+                                <th width="70%">URL</th>
```

Comment:
Wieso diese Änderung?

Reply:
Weil eine neue Spalte "Videos" in der Tabelle eingefügt wird (siehe Zeile 87 im PR). Die braucht die 5%.

Reply:
Omg, logisch. :)